### PR TITLE
Add `Create Team` button on /teams page

### PR DESCRIPTION
**Pull request by nisusam**
Screenshot for the https://github.com/exercism/exercism.io/issues/3274

create_team_button

### CODE REVIEW

**Comment by bernardoamc** [comment]
Awesome, thank you for the PR @nisusam! ❤️

File: public/sass/layouts/team_search_page.scss [hunk 1]
```diff
@@ -2,6 +2,11 @@
   margin-top: 20px;
 }
 
+.create-team-btn {
+  margin-top: 2.0em;
+  margin-bottom: .71428571em;
```
Comment (Insti) [inline public/sass/layouts/team_search_page.scss]: Is there a reason for so many significant digits here?

Comment (nisusam) [inline public/sass/layouts/team_search_page.scss]: @Insti, The same `margin-bottom` value is used for `Search for public teams` H3 text.

File: public/sass/layouts/team_search_page.scss [hunk 2]
```diff
@@ -3,8 +3,7 @@
 }
 
 .create-team-btn {
-  margin-top: 2.0em;
-  margin-bottom: .71428571em;
+  margin-top: 2em;
```
Comment (Insti) [inline public/sass/layouts/team_search_page.scss]: :tada: it's always good to get rid of redundant code.

**Comment by kytrinyx** [comment]
Thank you so much, @nisusam!

**Pull request merged**